portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+
+/** A comment. */
+public final class Comment {
+
+  private final long id;
+  private final String username;
+  private final String comment;
+  private final String imageId;
+  private final Date date;
+
+  public Comment(long id, String username, String comment, String imageId, Date date) {
+    this.id = id;
+    this.username = username;
+    this.comment = comment;
+    this.imageId = imageId;
+    this.date = date;
```

Comment:
Alternatively to my comment above, if Date is opaque to the user and cannot be modified (always sets to new Date()), you could remove it as a parameter and set it here only, this.date = new Date().

Reply:
+1, if the user is not allowed to manipulate the date, then it doesn't need to be provided as a parameter and can always be set to a new Date() object.

Reply:
I see, thing is that I'm using this constructor with date as a parameter, because when retrieving comments from Datastore, I create Comments objects from Datastore Entities, so I need to pass the date when they were created.